pkg/globalnet/controllers/ipam/globalnet.go

```diff
@@ -0,0 +1,47 @@
+package ipam
+
+import (
+	k8sv1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+)
+
+func (i *Controller) syncPodRules(podIP, globalIP string, addOrDelete Operation) {
```

Comment:
Since the options are only add or delete, I think it makes sense to use a `bool` here instead of `Operation`. Same with `syncServiceRules`.

Reply:
Done